`cargo build --release` throwing error E0599

### What version of bevy_dioxus are you using ?

1.0.0 or main(commit: ff428b75137fc814fb1aa8c0719808e4a5afb784)

### Does this issue reproduce with the latest release?

yes

### What operating system and processor architecture are you using ?

```
rustup show      -> stable-aarch64-apple-darwin (default)
rustc --version  -> rustc 1.61.0 (fe5b13d68 2022-05-18)
cargo --version  -> cargo 1.61.0 (a028ae42f 2022-04-29)
```

### What did you do?

Running `cargo build --release` will result in an error.

system

```cargo.toml
[package]
name = "bd_test"
version = "0.1.0"
edition = "2021"

[dependencies]
bevy = "0.7.0"
bevy_dioxus = { git = "https://github.com/JunichiSugiura/bevy_dioxus", branch = "main" } // Same for "1.0.0"
dioxus = "0.2.4"
```

`main.rs` is the same as `example/counter.rs.`

```
$ tree
.
├── Cargo.lock
├── Cargo.toml
└── src
    └── main.rs

1 directory, 3 files
```

### What did you expect to see?

Pass the build command

### What did you see instead?

```
   Compiling bevy_dioxus_desktop v0.1.0 (https://github.com/JunichiSugiura/bevy_dioxus?branch=main#ff428b75)
error[E0599]: no method named `open_devtools` found for struct `WebView` in the current scope
   --> /Users/arasan01/.cargo/git/checkouts/bevy_dioxus-7100cf5a55c8f3e2/ff428b7/packages/desktop/src/runner.rs:351:55
    |
351 | ...                   dioxus_window.webview.open_devtools();
    |                                             ^^^^^^^^^^^^^ method not found in `WebView`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `bevy_dioxus_desktop` due to previous error
warning: build failed, waiting for other jobs to finish...
```

--- a/packages/desktop/src/runner.rs
+++ b/packages/desktop/src/runner.rs
@@ -347,7 +347,9 @@ where
                                 }
                             }
                             WindowEvent::DevTool => {
+                                #[cfg(any(debug_assertions, feature = "devtools"))]
                                 let dioxus_window = dioxus_windows.get_mut(id).unwrap();
+                                #[cfg(any(debug_assertions, feature = "devtools"))]
                                 dioxus_window.webview.open_devtools();
                             }
                             WindowEvent::Eval(code) => {

